src/amo/components/LandingAddonsCard/index.js

```diff
@@ -13,10 +13,17 @@ export default class LandingAddonsCard extends React.Component {
     footerLink: PropTypes.object.isRequired,
     footerText: PropTypes.string.isRequired,
     header: PropTypes.node.isRequired,
+    loading: PropTypes.boolean,
+  }
+
+  static defaultProps = {
+    loading: false,
```

Comment:
Why is this needed? Don't we usually default to loading being false in the reducer?